packages/redis/src/index.js

```diff
@@ -0,0 +1,188 @@
+import Redis from 'ioredis';
+import shortId from 'shortid';
+
+export default class {
+  constructor(options) {
+    this.redis = new Redis(options);
+  }
+  async addUserToRoles(userId, roles, group) {
+    const roleIds = [];
+    let groupId;
+    await Promise.all(roles.map(async (role) => {
```

Comment:
What about `try`/`catch`? Is there any error handling needed for these async calls?